Formula/tailscale.rb

```diff
@@ -0,0 +1,35 @@
+class Tailscale < Formula
+  desc "Easiest, most secure way to use WireGuard and 2FA"
+  homepage "https://tailscale.com"
+  url "https://github.com/tailscale/tailscale.git",
+      tag:      "v1.16.2",
+      revision: "b56ba20549aedc0e56d09dca2552ca62421b2cbd"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    vars=Utils.safe_popen_read("./build_dist.sh", "shellvars")
+    ldflags = %W[
+      -s -w
+      -X tailscale.com/version.Long=#{vars.match(/VERSION_LONG=\"(.*)\"/)[1]}
+      -X tailscale.com/version.Short=#{vars.match(/VERSION_SHORT=\"(.*)\"/)[1]}
+      -X tailscale.com/version.GitCommit=#{vars.match(/VERSION_GIT_HASH=\"(.*)\"/)[1]}
+    ].join(" ")
+    system "go", "build", *std_go_args(ldflags: ldflags), "tailscale.com/cmd/tailscale"
+    system "go", "build", *std_go_args(ldflags: ldflags), "-o", bin/"tailscaled", "tailscale.com/cmd/tailscaled"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/tailscale version")
```

Comment:
```suggestion
    assert_equal version.to_s, shell_output("#{bin}/tailscale version").chomp
```

Reply:
I don't think that would work with this format
```
1.16.2
  tailscale commit: b56ba20549aedc0e56d09dca2552ca62421b2cbd-dirty
  go version: go1.17.2
```

Reply:
Do you want to
```ruby
(buildpath/".brew_home").rmtree
```
or change your regex to
```
VERSION_GIT_HASH="(.*)-dirty"
```
to get rid of the `-dirty` part in the commit?

We may also want to do an `assert_match` for `/[a-f0-9]{40}/` to make sure we picked up the commit hash.